src/services/export-api-client.js

```diff
@@ -0,0 +1,40 @@
+import HttpClient from './http-client.js';
+
+const mockDocuments = [
```

Comment:
On the other hand, if we leave it here, eslint won't allow us to not use it. In another file it's easy to forget to delete it.

Reply:
I'd leave it here close to the usage, because it's supposed to be used just once and it's a playground. We can always add mock data that is intended to stay if and when we'll need to.

Reply:
Okay, mock-data.delete.me.js :)). Bad idea. I thought it would kinda clear some of the pollution that this brings in the meanwhile by just importing a constant here and in the other file where mock data is used.

Reply:
For me it would move the pollution under the rug, cause we'll end up with an unused mock data. No worries, this will not stay long unless we decide to not work on importing anymore.

Reply:
Don't know, for me it's easier to delete a file somewhere when we want to remove the mocks and have broken links in the code that tell me where it was used. I see it more like putting all the garbage in one place so we can take it out easier. This way you need to remember where else did you have mock data on the live code (which gets harder for us since we didn't write the code). But again, it was just an idea :).

Reply:
Importing has more changes of forgetting it, from my point. It's not about ease of deletion, deletion is easy anyway.
I'd rather not extract mock data used temporarily and juts in one place.